internal/controllers/clusterctl/config.yaml

```diff
@@ -11,12 +11,12 @@ data:
     providers:
     # Cluster API core provider
     - name:         "cluster-api"
-      url:          "https://github.com/rancher-sandbox/cluster-api/releases/v1.8.4/core-components.yaml"
+      url:          "https://github.com/rancher-sandbox/cluster-api/releases/v1.9.0/core-components.yaml"
       type:         "CoreProvider"
 
     # Infrastructure providers
     - name:         "aws"
-      url:          "https://github.com/rancher-sandbox/cluster-api-provider-aws/releases/v2.6.1/infrastructure-components.yaml"
+      url:          "https://github.com/rancher-sandbox/cluster-api-provider-aws/releases/v2.7.1/infrastructure-components.yaml"
```

Comment:
I didn't find a GH release for 2.7.1 in https://github.com/rancher-sandbox/cluster-api-provider-aws/releases should we create one?

Reply:
It gets created automatically when someone trigger an image build for this release

Reply:
@yiannistri here it is: https://github.com/rancher-sandbox/cluster-api-provider-aws/releases/tag/v2.7.1